src/main/scala/Hangman.scala

```diff
@@ -0,0 +1,94 @@
+object Hangman extends App{
+  import scala.io.Source
+
+  val fileName = "C:\\Users\\Admin\\scala\\exercises\\words.txt"
+  val availableWords = Source.fromFile(fileName).getLines.toArray
+  val wordPosition = scala.util.Random.nextInt(availableWords.length)
+  val chosenWord = availableWords(wordPosition)
+  var spaces = Array.fill(chosenWord.length){""}
+  var letter = ""
+  var wrongGuesses = 0
+  var correctGuesses = 0
+  var guessStorage = Array.fill(26){""}
+  var noOfGuesses = 0
+
+  def underscores(word:String): Unit ={
+    for(i <- 0 until word.length){
+     spaces(i) = "_ "
+     print(spaces(i))
+    }
+    println()
+  }
+
+  def guess(choice:String): Unit ={
```

Comment:
too many things are happening in this method, split it up into multiple helper ones